template/docs/_static/pybamm.css

```diff
@@ -0,0 +1,230 @@
+CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+*/
+
+/* Font */
+
+@import url("https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,400;0,700;0,900;1,400;1,700;1,900&family=Open+Sans:ital,wght@0,400;0,600;1,400;1,600&display=swap");
+
+body {
+  font-family: "Open Sans", sans-serif;
+}
+
+/* Main page overview cards */
+
+.sd-card {
+  background: #fff;
+  border-radius: 0;
+  padding: 30px 10px 20px 10px;
+  margin: 10px 0px;
+}
+
+.sd-card .sd-card-header {
+  text-align: center;
+}
+
+.sd-card .sd-card-header .sd-card-text {
+  margin: 0px;
+}
+
+.sd-card .sd-card-img-top {
+  height: 52px;
+  width: 52px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.sd-card .sd-card-header {
+  border: none;
+  background-color: white;
+  color: #150458 !important;
+  font-size: var(--pst-font-size-h5);
+  font-weight: bold;
+  padding: 2.5rem 0rem 0.5rem 0rem;
+}
+
+.sd-card .sd-card-footer {
+  border: none;
+  background-color: white;
+}
+
+.sd-card .sd-card-footer .sd-card-text {
+  max-width: 220px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+/* Dark theme tweaking */
+
+html[data-theme="dark"] .sd-card img[src*=".svg"] {
+  filter: invert(0.82) brightness(0.8) contrast(1.2);
+}
+
+/* Main index page overview cards */
+
+html[data-theme="dark"] .sd-card {
+  background-color: var(--pst-color-background);
+}
+
+/* Inheritance diagram dropdown options */
+
+html[data-theme="dark"] .sd-card .sd-card-header {
+  background-color: var(--pst-color-background);
+  color: #459db9 !important;
+}
+
+html[data-theme="dark"] .sd-card .sd-card-footer {
+  background-color: var(--pst-color-background);
+}
+
+details.sd-dropdown {
+  box-shadow: none !important;
+}
+
+.sd-summary-content blockquote {
+  border-left: 0 !important;
+}
+
+/* Overrides for sphinx-hoverxref since it does not support a native dark theme, see */
+/* https://github.com/readthedocs/sphinx-hoverxref/issues/231 */
+
+/* These will ensure that the tooltip inherits the PyData colours */
+
+.tooltipster-sidetip.tooltipster-shadow .tooltipster-box {
+  background: var(--pst-color-background) !important;
+}
+
+.tooltipster-sidetip.tooltipster-shadow .tooltipster-arrow {
+  background: none !important;
+  overflow: hidden;
+  position: absolute;
+}
+
+.tooltipster-sidetip.tooltipster-shadow.tooltipster-right
+  .tooltipster-arrow-border {
+  border-right-color: var(--pst-color-background) !important;
+}
+
+.tooltipster-sidetip.tooltipster-shadow.tooltipster-left
+  .tooltipster-arrow-border {
+  border-left-color: var(--pst-color-background) !important;
+}
+
+/* Custom styles for Algolia DocSearch v3 items and modals */
+/* to make them congruous with PST on light and dark themes */
+
+@import url("https://cdn.jsdelivr.net/npm/@docsearch/css@3");
```

Comment:
Afaik all `@import` should be on top in a css, is it not giving any issue because it's not?

Reply:
I just imported this CSS from pybamm just to fill in the `_static` directory discussed [here](https://github.com/pybamm-team/pybamm-cookiecutter/pull/39#discussion_r1704193154), which was failing the doctests. Without any content inside the directory, it wouldn't copy and fail the tests again so Ferran just added a quick fix by adding the CSS file so it would copy and not fail the doctests. The file is unnecessary and we could maybe replace it with something else to have something inside the `_static` directory.

Reply:
it's not on top, but never mind as this file have been copied from pybamm as it is.